cmd/bridge/main.go

```diff
@@ -54,6 +54,9 @@ const (
 
 	// Well-known location of the GitOps service. This is only accessible in-cluster
 	openshiftGitOpsHost = "cluster.openshift-pipelines-app-delivery.svc:8080"
+
+	// Well-known location of the Devfile service. This is only accessible in-cluster
+	openshiftDevfileParserHost = "cluster.openshift-devfile-app-delivery.svc:8080"
```

Comment:
Can't find where this is used. Can we remove it?

Reply:
Yeah, I don't understand what this was for... so I'll remove it and test around and see if I see an issue. 👍